internal/publish/publish.go

```diff
@@ -444,3 +449,9 @@ func (h *PublishHandler) process(ctx context.Context, data *verifyapi.Publish, b
 		},
 	}
 }
+
+// chaffPushResponse takes a chaffing string, and builds a chaff response.
+func chaffPublishResponse(s string) interface{} {
+	return verifyapi.PublishResponse{Padding: s[:(len(s)*7)/10]}
```

Comment:
the chaff library already accounts for base64 encoding

Reply:
and doing a sub-slice here might make this invalid base64

Reply:
I didn't realize anyone was ever going to decode the chaff. In any case, removed.